test/lib/utils/decoder_test.rb

```diff
@@ -0,0 +1,74 @@
+require "test_helper"
+require "jwt"
+require "openssl"
+require "json"
+require "tempfile"
+require_relative "../../../app/lib/utils/decoder_factory"
```

Comment:
it seems like these would be autoloaded -- can you try removing?

Reply:
i couldn't get this to work.  I always get this:
```
NameError: uninitialized constant SecretDecoderTest::SecretDecoder
```

Reply:
It works okay for me when they are removed